File: gcd_euclids.c
#include<stdio.h>
#include<time.h>
int main()
{
    clock_t t;
    t = clock();

    int m, n, r;
    printf("Enter the bigger number : ");
    scanf("%d", &m);
    printf("Enter the smaller number : ");
    scanf("%d", &n);
    while(n!=0)
    {
        r = m % n;
        m = n;
        n = r;
    }
    printf("GCD is : %d", m);
    t = clock() - t;
    printf("\nExecution time in seconds : %f", ((double)t/CLOCKS_PER_SEC));
    return 0;
}

File: gcd_euclids.py
import time
start = time.time()

num1 = int(input("Enter the bigger number : "))
num2 = int(input("Enter the smaller number : "))

while(num2!=0):
    rem = num1 % num2
    num1 = num2
    num2 = rem

print("GCD is given by : ", num1)
print("%s seconds" % (time.time() - start))